src/Microsoft.AspNetCore.Razor.Language/DefaultRazorProjectEngine.cs

```diff
@@ -65,7 +66,7 @@ protected override RazorCodeDocument CreateCodeDocumentCore(RazorProjectItem pro
 
             var importFeature = GetRequiredFeature<IImportProjectFeature>();
             var importItems = importFeature.GetImports(projectItem);
-            var importSourceDocuments = importItems.Select(ConvertToSourceDocument);
+            var importSourceDocuments = importItems.Select(i => ConvertToSourceDocument(i));
```

Comment:
revert this YO

Reply:
Nope. It won't accept the new signature.

Reply:
O_O oooo default params 😢